tmt/steps/report/polarion.py

```diff
@@ -209,7 +209,64 @@ class ReportPolarionData(tmt.steps.report.ReportStepData):
 @tmt.steps.provides_method('polarion')
 class ReportPolarion(tmt.steps.report.ReportPlugin[ReportPolarionData]):
     """
-    Write test results into an xUnit file and upload to Polarion
+    Write test results into an xUnit file and upload to Polarion.
+
+    *As a tester I want to review tests in Polarion
+    and have all results linked to existing test cases there.*
+
+    In order to get quickly started create a pylero config
+    file ``~/.pylero`` in your home directory with the
+    following content:
+
+    .. code-block:: ini
+
+        [webservice]
+        url=https://{your polarion web URL}/polarion
+        svn_repo=https://{your polarion web URL}/repo
+        default_project={your project name}
+        user={your username}
+        password={your password}
+
+    See the ``Pylero Documentation`` for more details on how
+    to configure the ``pylero`` module.
+
+    https://github.com/RedHatQE/pylero
```

Comment:
```suggestion
    __ https://github.com/RedHatQE/pylero
```

Why not keeping the anonymous link? Was this an intentional change? Perhaps because of the `--help` rendering?

Reply:
@psss yes, I tried that but it fails when generating the docs:
`Failed to import the 'tmt.steps.report.polarion' module from '/home/mcasquer/Workspace/root_tmt/docs'.`